src/main/java/baritone/bot/utils/ExampleBaritoneControl.java

```diff
@@ -177,6 +178,10 @@ public void onSendChatMessage(ChatEvent event) {
                 }
             }
         }
+        if(msg.toLowerCase().equals("map")) {
+            MapChunk chunk = new MapChunk(world().getChunk(player().getPosition()));
```

Comment:
We use `playerFeet()` instead of `player().getPosition()` everywhere else because for some strange reason, `getPosition()` adds 0.5 to x, y, and z before casting to integer. `playerFeet` gives the exact block the player sees when you look straight down, `getPosition` will give the one that's next to it if you're in the top (coordinate-wise) half of the block.

Reply:
This is _purely_ for testing. You're right that it's not great, but this will be removed in a later update.